### PR TITLE
New package: SuperCuber.Dotter version 0.13.1

### PR DESCRIPTION
**Pull request by FlippingBinary**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/131412)

### CODE REVIEW

**Comment by FlippingBinary** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/s/SuperCuber/Dotter/0.13.1/SuperCuber.Dotter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SuperCuber.Dotter
+PackageVersion: 0.13.1
+InstallerType: portable
+Commands:
+- dotter
```
Comment (stephengillie) [inline manifests/s/SuperCuber/Dotter/0.13.1/SuperCuber.Dotter.installer.yaml:8]:
```suggestion
- dotter
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Comment (FlippingBinary) [reply manifests/s/SuperCuber/Dotter/0.13.1/SuperCuber.Dotter.installer.yaml:8]: Oh wow! I had no idea winget could specify dependencies like that. That would be perfect!

**Comment by FlippingBinary** [comment]
The GitHub repository is `SuperCuber/dotter`, so should the package name actually be `SuperCuber.dotter` instead of `SuperCuber.Dotter`?

**Comment by stephengillie** [comment]
> The GitHub repository is `SuperCuber/dotter`, so should the package name actually be `SuperCuber.dotter` instead of `SuperCuber.Dotter`?

Yes, and this would also match the EXE's name.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**